src/v2i-hub/CARMACloudPlugin/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-PROJECT ( CARMACloudPlugin VERSION 3.0.0 LANGUAGES CXX )
+PROJECT ( CARMACloudPlugin VERSION 7.3.1 LANGUAGES CXX )
```

Comment:
Technically don't we want to update these version numbers as well. @jwilliammartin you created a confluence document to capture everywhere this version number needs to be changed. We may want to do that as well.

Reply:
Found it https://usdot-carma.atlassian.net/wiki/spaces/V2XH/pages/2375221249/Version+Number+Update. We may want to make these version numbers reflect 7.4.0